packages/amplify-cli/src/extensions/amplify-helpers/docs-manager.ts

```diff
@@ -0,0 +1,22 @@
+import fs from 'fs-extra';
+const ReadMeContent = `# Getting Started with amplify-cli
+This directory was generated with [amplify-cli][] command. This README helps you to find resources you are looking for.
```

Comment:
Amplify CLI [docs.amplify.aws/cli]

Reply:
I'd shorten this section to `This directory was generated by [Amplify CLI][docs.amplify.aws/cli] command.`

Reply:
Great input, I've updated :)